src/boot_O2_g3/idle.c

```diff
@@ -110,10 +115,11 @@ void Idle_InitVideo(void) {
 
 void Idle_ThreadEntry(void* arg) {
     Idle_InitVideo();
-    osCreatePiManager(150, &gPiMgrCmdQ, sPiMgrCmdBuff, ARRAY_COUNT(sPiMgrCmdBuff));
+    osCreatePiManager(OS_PRIORITY_PIMGR, &gPiMgrCmdQueue, sPiMgrCmdBuff, ARRAY_COUNT(sPiMgrCmdBuff));
     StackCheck_Init(&sMainStackInfo, sMainStack, STACK_TOP(sMainStack), 0, 0x400, "main");
     osCreateThread(&sMainThread, Z_THREAD_ID_MAIN, Main_ThreadEntry, arg, STACK_TOP(sMainStack), Z_PRIORITY_MAIN);
     osStartThread(&sMainThread);
```

Comment:
So the reason some `Idle_` functions but not all were renamed to `Main_` is because those functions that were renamed run on the Main thread created here? So `Idle_ThreadEntry` remains `Idle_` here creates the main thread but runs on the Idle thread. And functions that run on the main thread are named `Main_`?

Reply:
Correct, `Idle` only has the one helper function, the rest end up running on the `Main` thread.